pkg/karmadactl/unjoin.go

```diff
@@ -271,6 +271,7 @@ func deleteClusterObject(controlPlaneKarmadaClient *karmadaclientset.Clientset,
 
 	err := controlPlaneKarmadaClient.ClusterV1alpha1().Clusters().Delete(context.TODO(), opts.ClusterName, metav1.DeleteOptions{})
 	if apierrors.IsNotFound(err) {
+		fmt.Printf("Error:No cluster object %s found in karmada control Plane\n", opts.ClusterName)
```

Comment:
We could directly delete these error check `apierrors.IsNotFound(err)`.

Reply:
> We could directly delete these error check `apierrors.IsNotFound(err)`.

I think, deleting this error check apierrors.IsNotFound(err) would skip the stage of deleting the cluster role, cluster rolebindings and service account from the unjoining cluster. Only cluster object in karmada control Plane has been  deleted, in some situation, it's needed to re-execute unjoin command to delete the above-mentioned resources of unjoining cluster.

Reply:
>  Only cluster object in karmada control Plane has been deleted, in some situation, it's needed to re-execute unjoin command to delete the above-mentioned resources of unjoining cluster.

Ok, we can replace `fmt.Printf` with `klog`?

Reply:
> > Only cluster object in karmada control Plane has been deleted, in some situation, it's needed to re-execute unjoin command to delete the above-mentioned resources of unjoining cluster.
> 
> Ok, we can replace `fmt.Printf` with `klog`?

I prefer "fmt.Printf" , because it is more official and eye-catching than "klog".

Reply:
Mainly it is not a actural error, only a promption. Generally, if report `Error: `, the command is stoped immidetely.

Reply:
> 

I agree with you. Should I remove the "Error:" and use "fmt.Printf"?

Reply:
> Mainly it is not a actural error, only a promption. Generally, if report `Error: `, the command is stoped immidetely.

I agree with you. Should I remove the "Error:" and use "fmt.Printf"?

Reply:
> Mainly it is not a actural error, only a promption. Generally, if report `Error: `, the command is stoped immidetely.

I agree with you.

Reply:
I was going to replace all the `klogs` in cmd with `fmt` before, but it wasn't all done. If we use `fmt` only at this point in `unjoin cmd`, the format would be strange.

Reply:
> 

yeah, I've replace it with klog.